vtysh: don't send access list commands to staticd

rzalamena opened the pull request:
Summary
------------

Recent northbound implementation increased the pressure on libyang on all daemons with big configurations, so by not sending some commands to daemons that do not use them should help us save CPU cycles / memory usage.

rzalamena commented:
Rebased with latest code

donaldsharp commented:
ci:rerun

rzalamena commented:
CI:rerun

rwestphal commented on vtysh/extract.pl.in:
```diff
@@ -98,7 +98,10 @@ sub scan_file {
         }
-        elsif ($file =~ /lib\/(filter|filter_cli|lib_vty)\.c$/) {
+        elsif ($file =~ /lib\/(filter|filter_cli)\.c$/) {
+            $protocol = "VTYSH_RMAP";
```
ldpd uses ACLs but not route-maps :(

I think you'll need to add a new group of daemons like `VTYSH_FILTER`.

rzalamena replied:
@rwestphal thanks, I refactored the commits and shortened the daemon list to send commands to one by one.

Also I alphabetically ordered the list so it is easier to spot the missing entry.

rzalamena commented:
I rebased the PR and refactored the implementation:

- First commit adds access list command category to all daemons
- Second and forth commit removes `sharpd` and `staticd` from the list of daemons to send command to from access list and route maps
